ethereum_txsigner/js/mascara/Readme.md

```diff
@@ -0,0 +1,92 @@
+# Metamask Mascara wrapper
```

Comment:
Shouldn't this be a stand-alone JS library (in a separate github repo) that the module would require?